pkg/noobaaaccount/noobaaaccount.go

```diff
@@ -374,6 +388,48 @@ func RunRegenerate(cmd *cobra.Command, args []string) {
 
 }
 
+// RunCredentials runs a CLI command
+func RunCredentials(cmd *cobra.Command, args []string) {
+	log := util.Logger()
+
+	if len(args) != 1 || args[0] == "" {
+		log.Fatalf(`❌ Missing expected arguments: <noobaa-account-name> %s`, cmd.UsageString())
+	}
+
+	name := args[0]
+
+	o := util.KubeObject(bundle.File_deploy_crds_noobaa_io_v1alpha1_noobaaaccount_cr_yaml)
+	noobaaAccount := o.(*nbv1.NooBaaAccount)
+	noobaaAccount.Name = name
+	noobaaAccount.Namespace = options.Namespace
+
+	var accessKeys nb.S3AccessKeys
+
+	accessKeys.AccessKey, _ = cmd.Flags().GetString("access_key")
+	accessKeys.SecretKey, _ = cmd.Flags().GetString("secret_key")
```

Comment:
What do you think about using `util.GetFlagStringOrPromptPassword` to avoid this printing in the shell? for example:
https://github.com/noobaa/noobaa-operator/blob/fcc5240f5af690596823f2ab63e9debd9a640fda/pkg/namespacestore/namespacestore.go#L447-L448

Reply:
Yes, I think it is a good idea to get keys using prompt and also not printing them on terminal. Thanks, will update it.